docs/releases/4.0.txt

```diff
@@ -325,6 +325,17 @@ Miscellaneous
   to ``reorder_tests()``. It now accepts an iterable of tests rather than a
   test suite, and returns an iterator of tests.
 
+* Values in the setting :setting:`CSRF_TRUSTED_ORIGINS` setting must include
+  the scheme (``'http://'`` or ``'https:://'``) instead of only the hostname.
+  Values that started with a dot, must now also include an asterisk before the
+  dot. For example, change ``'.example.com'`` to ``'https://*.example.com'``.
+
+* As CSRF protection now consults the ``Origin`` header, you may need to set
+  :setting:`CSRF_TRUSTED_ORIGINS`, particularly if you allow requests from
+  subdomains by setting :setting:`CSRF_COOKIE_DOMAIN` (or
+  :setting:`SESSION_COOKIE_DOMAIN` if :setting:`CSRF_USE_SESSIONS` is enabled)
+  to a value starting with a dot.
+
```

Comment:
This change will effect quite a lot of people (≈everyone?). I think this could be pulled up to it's own section in `Backwards incompatible changes in 4.0`, plus being linked to from the entry in `CSRF`. 

I know people will get the System Check on first run, but it's kind of a breaking change, so a bit more highlighting maybe?

Reply:
done

Reply:
Super. Thanks @timgraham — that looks great. I will look at the code changes today. 👍